### ISSUE TITLE
No render output when using plasma on sub model and default render style

### ISSUE THREAD

**Issue by ptr727** [xLightsSequencer/xLights#5051]
**Describe the bug**
- Opened a working 2023 project in 2024.19.0
- Open sequence, play, works fine
- Click re-render, and half of prop is not rendering
- Per help on zoom room the issue was that plasma on submodel using default render style does not work
- Change render style to per preview and render works again
- Also tested using `xLights64_2024_19_nightly-f878a35` nightly same behavior

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)
1. Add plasma to sub model
2. Use default render style
3. No render output

**Expected behavior**
Default render style works 

**Screenshots**
- Attached

**Versions (please complete the following information):**
 - OS: Win11
 - 2024.19.0

**Additional context**
Zoom room asked me to log a bug.

**Attachments**
- Project attached.

![image](https://github.com/user-attachments/assets/f0a8d15d-67da-4219-9bbb-71c282432216)
![image](https://github.com/user-attachments/assets/9c3199f4-55c7-42ae-8d10-6a2e5e76fb68)
[mrjingles.zip](https://github.com/user-attachments/files/18088638/mrjingles.zip)

**Comment by derwin12** [comment]
Curious if this worked in prior versions ? 2023.x ?

**Comment by ptr727** [comment]
> Curious if this worked in prior versions ? 2023.x ?

Yes, last rendered in v2023 for xmas '23, even if I open the project folder in 2024.19 and just press play the render cache content plays, as soon as I re-render the plasma effects disappear.

**Comment by derwin12** [comment]
Would you mind just doing tools->package sequence with your test sequence and attach that here, so we can run it thru the debugger.

**Comment by ptr727** [comment]
[Mr Jingles.zip](https://github.com/user-attachments/files/18105007/Mr.Jingles.zip)

**Comment by derwin12** [comment]
The default seems to be working in 2025-04. Likely correcting in the move to ISPC coding.
